lib/Ray/include/Surface.hpp

```diff
@@ -0,0 +1,28 @@
+/*
+** EPITECH PROJECT, 2021
+** Bomberman
+** File description:
+** Surface
+*/
+
+#ifndef SURFACE_HPP_
+#define SURFACE_HPP_
+
+#include "Drawables/Drawable.hpp"
+
+namespace Ray {
+    class Surface {
```

Comment:
What is a surface? Is it like a canvas?

Reply:
Yes, it is a surface where shapes can be drawn on, such as a window, an Image, ...

Reply:
A Surface is often a synonym for texture (SDL2 had surfaces)

Reply:
How should it be called then?